package.json

```diff
@@ -69,7 +70,8 @@
     "sinon-sandbox": "^1.0.2"
   },
   "peerDependencies": {
-    "react": ">=0.14"
+    "react": ">=0.14",
+    "prop-types": ">=15.5.6"
```

Comment:
Interesting, I was also thinking `prop-types` would be best as a peerDeps, since it will be used by almost all react projects.
Any reasons to let each projects potentially use different version of it? It shouldn't cause any issues, but still...